app/code/Magento/CustomerGraphQl/Model/Resolver/CreateCustomer.php

```diff
@@ -55,6 +72,14 @@ public function resolve(
             throw new GraphQlInputException(__('"input" value should be specified'));
         }
 
+        if (!$this->scopeConfig->getValue(
```

Comment:
Please, move this logic to `\Magento\CustomerGraphQl\Model\Customer\CreateCustomerAccount::createAccount`

Reply:
btw @sedonik,

What do you think about extracting this logic to separate class in ` Magento\Newsletter\Model` namespace?
Because now we have got some duplication of getting config value in this class and `\Magento\Newsletter\Observer\PredispatchNewsletterObserver`

We can encapsulate it.

Reply:
@naydav, thank you for the response.

I can create class Magento\Newsletter\Model\Config with method "isActive" which returns "newsletter/general/active" value.

What do you think, am I correct?